packages/gamut-labs/src/landingPage/Feature.tsx

```diff
@@ -1,30 +1,81 @@
-import { Markdown } from '@codecademy/gamut';
+import { Box, Markdown } from '@codecademy/gamut';
 import { mediaQueries } from '@codecademy/gamut-styles';
 import styled from '@emotion/styled';
 import React from 'react';
 
 import { Text } from '../experimental/Text';
 import { BaseProps } from './types';
 
-const Icon = styled.img`
-  width: 4rem;
-  margin-top: 3rem;
-  margin-bottom: 2rem;
-`;
+export type FeaturedImageProps = {
+  src: string;
+  alt: string;
+};
+export const FeaturedImage: React.FC<FeaturedImageProps> = ({ src, alt }) => (
+  <img
+    alt={alt}
+    src={src}
+    width="100%"
+    style={{ marginBottom: '2rem' }}
+    data-testid="feature-image"
+  />
+);
 
-const Image = styled.img`
-  width: 100%;
-  margin-top: 3rem;
-  margin-bottom: 2rem;
-`;
+export type FeaturedIconProps = {
+  src: string;
+  alt: string;
+};
+export const FeaturedIcon: React.FC<FeaturedIconProps> = ({ src, alt }) => (
+  <img
+    alt={alt}
+    src={src}
+    width="64px"
+    style={{ marginBottom: '2rem' }}
+    data-testid="feature-icon"
+  />
+);
 
-const StyledMarkdown = styled(Markdown)`
-  font-size: 1rem;
-  margin: 1rem 0 0;
-`;
+export const FeaturedStat: React.FC = ({ children }) => (
+  <Text
+    as="div"
+    fontSize={{ xs: 44, lg: 64 }}
+    fontWeight="title"
+    data-testid="feature-stat"
+  >
+    {children}
+  </Text>
+);
+
+export type FeaturedTitleProps = {
+  as?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+};
+
+export const FeaturedTitle: React.FC<FeaturedTitle> = ({ as, children }) => (
+  <Text
+    as={as || 'h3'}
+    marginBottom={0}
+    fontSize={{ xs: 22, lg: 26 }}
+    fontWeight="title"
+  >
+    {children}
+  </Text>
+);
+
+export type FeaturedDescriptionProps = Pick<
+  BaseProps,
+  'desc' | 'onAnchorClick'
+>;
+export const FeaturedDescription: React.FC<FeaturedDescriptionProps> = ({
+  desc,
+  onAnchorClick,
+}) => (
+  <Box marginBottom={16} fontSize={16}>
+    <Markdown text={desc} onAnchorClick={onAnchorClick} />
+  </Box>
```

Comment:
i'm still seeing the font size as 17.6 which is the default for markdown. poked around a little bit and it looks like we can add `spacing="none"` to `Markdown` which will reduce the font size to 16 and add 1rem of padding to the bottom. thoughts? otherwise i think we need to set the font size directly on the `Markdown` component
```suggestion
    <Markdown text={desc} spacing="none" onAnchorClick={onAnchorClick} />
```

Reply:
hey, i'm all for simplifying.  fixing.